packages/components/modal/tests/modal.test.tsx

```diff
@@ -305,4 +305,152 @@ describe("<Modal />", () => {
       "justify-content": "center",
     })
   })
+
+  interface AnimationProps {
+    animation?: "bottom" | "left" | "none" | "right" | "scale" | "top"
+    duration?: number
+  }
+
+  const ModalAnimationExample = ({ animation, duration }: AnimationProps) => {
```

Comment:
@nathanlao 
Instead of defining your own `AnimationProps`, it is better to pick the properties you need from `ModalProps`.

```tsx
const ModalAnimationExample = ({ animation, duration }: Pick<ModalProps, "animation" | "duration">) => {
  // ...
}
```

Reply:
thank you for the suggestion!